src/plugins/dataflow-tool/components/dataflow-program.tsx

```diff
@@ -1123,7 +1123,7 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
 
       // update virtual sensors
       if (chInfo?.virtualValueMethod) {
-        const timeFactor = chInfo?.name === "EMG" ? 100 : 1000;
+        const timeFactor = chInfo?.name.match(/[EMG]|[FSR]/) ? 100 : 1000;
```

Comment:
Maybe `timeFactor` should just be specified in the `NodeChannelInfo`. I feel like that would be cleaner than determining it from complex rules based on the name that are only going to get more worse moving forward.

Reply:
That's a good idea.  Made the [update](https://github.com/concord-consortium/collaborative-learning/pull/1326/commits/cca1b89297a6d2b0fbe7cb59b025e3ab194151cf)